doc/introduction/importing_workflows.rst

```diff
@@ -0,0 +1,268 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing workflows
```

Comment:
I'd recommend a longer, more explicit title, as the word 'workflow' is quite generic. Perhaps 'Importing workflows from other quantum computing frameworks' or something similar? This would help the page to show up in more relevant web searches.

Reply:
Decided offline - let's treat the title as a separate question as a longer one can look not so great in the left side bar.